frontend/api.js

```diff
@@ -417,9 +415,15 @@ function getResults(str, libs, include_builtins, include_core, page) {
 
 	xmlHttp.open('GET', url, true); // true for asynchronous
 	xmlHttp.send(null);
-	if (document.location.hash.substring(1) != encodeURIComponent(str)) {
+	if (true || document.location.hash.substring(1) != encodeURIComponent(str)) {
```

Comment:
Why this change? We should not set `refresh_on_hash` to false when the request has not been changed, because this will cause unexpected page reloads.